electron/Transport.js

```diff
@@ -0,0 +1,14 @@
+const Transport = require('winston-transport');
+
+module.exports = class CustomTransport extends Transport {
+  constructor(opts) {
+    super(opts);
+    this.loggedMessages = opts.loggedMessages;
+  }
+
+  log(info, callback) {
+    this.loggedMessages.push(`${info.timestamp} [${info.level}]: ${info.message}`);
+    // Perform the writing to the remote service
+    callback();
+  }
+};
```

Comment:
It looks like maybe an old version of the `InMemoryTransport` got added back. Is this intentional? Or can we remove this file?

Reply:
The old version must have been added back in during rebasing. I've deleted it.